youtube_dl/extractor/dailymotion.py

```diff
@@ -182,13 +182,18 @@ def _real_extract(self, url):
                 if metadata_url:
                     metadata_url = metadata_url.replace(':videoId', video_id)
                 else:
-                    metadata_url = update_url_query(
-                        'https://www.dailymotion.com/player/metadata/video/%s'
-                        % video_id, {
-                            'embedder': url,
-                            'integration': 'inline',
-                            'GK_PV5_NEON': '1',
-                        })
+                    if (17 < age_limit) :
+                        metadata_url = update_url_query(
+                            'https://www.dailymotion.com/player/metadata/video/%s'
+                            % video_id, {})
```

Comment:
By far the better solution is to pull the extra `dict` out into a variable guarded by this `if` rather than duplicating the "meat" of that call; something like:

```python
                    if 17 < age_limit:
                        metadata_extra = {}
                    else:
                        metadata_extra = {
                                'embedder': url,
                                'integration': 'inline',
                                'GK_PV5_NEON': '1',
                        }
                    metadata_url = update_url_query(
                            'https://www.dailymotion.com/player/metadata/video/%s'
                            % video_id, metadata_extra)
```